Snakefile

```diff
@@ -38,7 +38,10 @@ config["scenario"]["unc"] = [
 ]
 
 load_data_paths = get_load_paths_gegis("data", config)
-COSTS = "data/costs.csv"
```

Comment:
We need to improve the snakefile; it's getting superlong.... but not now...
Also in the feature, we need to understand and give priorities to the features we want